File: jee/papers/2016/python_comp/codes/ee16b1029.m
clear;
close;
clc;

x = linspace(-10,10,100);
y = (1./(2.*x)).*log(1+(tan(x.^(1./2)).^2));

plot(x,y,'LineWidth',4,0,0.5,'ro','LineWidth',4);
xlabel("x");
ylabel("log(p)");
grid;
print ('ee16b1029.eps', '-color')

File: programming/EE1083/jee/2016/python_comp/codes/ee16b1029.py

import numpy as np
import matplotlib.pyplot as plt

x = np.linspace(0.01,10,100)
y = (1/(2*x))*(np.log(1 + (np.tan(np.sqrt(x)))**2))
plt.plot(x,y)
plt.grid()
plt.xlabel('x')
plt.ylabel('log(p)')
plt.plot(0,0.5,'o')
plt.savefig('../figs/ee16b1029.eps')
plt.show()


